code/modules/organs/internal/psionic.dm

```diff
@@ -44,7 +44,15 @@
 		/obj/item/organ/internal/psionic_tumor/proc/psychic_call,
 		/obj/item/organ/internal/psionic_tumor/proc/psychic_banish,
 		/obj/item/organ/internal/psionic_tumor/proc/journey_to_nowhere,
-		/obj/item/organ/internal/psionic_tumor/proc/psionic_armor
+		/obj/item/organ/internal/psionic_tumor/proc/psionic_armor,
+		/obj/item/organ/internal/psionic_tumor/proc/kinetic_blaster,
+		/obj/item/organ/internal/psionic_tumor/proc/cryo_kinetic_blaster,
+		/obj/item/organ/internal/psionic_tumor/proc/pyro_kinetic_blaster,
+		/obj/item/organ/internal/psionic_tumor/proc/electro_kinetic_blaster,
+		/obj/item/organ/internal/psionic_tumor/proc/kinetic_barrier,
+		/obj/item/organ/internal/psionic_tumor/proc/chosen_control,
+		/obj/item/organ/internal/psionic_tumor/proc/detect_thoughts,
+		/obj/item/organ/internal/psionic_tumor/proc/psychoactive_manipulation
```

Comment:
Yo won't this give the new powers to all psions except psychiatrists?

Reply:
No, it is only for testing purposes